common/src/main/java/org/vivecraft/mixin/client/renderer/GameRendererVRMixin.java

```diff
@@ -857,167 +887,6 @@ public Matrix4f getThirdPassProjectionMatrix() {
 		return thirdPassProjectionMatrix;
 	}
 
-	@Override
-	public void drawFramebufferNEW(float partialTicks, boolean renderWorldIn, PoseStack matrixstack) {
-		if (!this.minecraft.noRender) {
-			Window window = this.minecraft.getWindow();
-			Matrix4f matrix4f = Matrix4f.orthographic(0.0F, (float) ((double) window.getWidth() / window.getGuiScale()),
-					0.0F, (float) ((double) window.getHeight() / window.getGuiScale()), 1000.0F, 3000.0F);
-			RenderSystem.setProjectionMatrix(matrix4f);
-			PoseStack posestack = RenderSystem.getModelViewStack();
-			posestack.pushPose();
-			posestack.setIdentity();
-			posestack.translate(0.0D, 0.0D, -2000.0D);
-			RenderSystem.applyModelViewMatrix();
-			Lighting.setupFor3DItems();
-			PoseStack posestack1 = new PoseStack();
-
-			int i = (int) (this.minecraft.mouseHandler.xpos() * (double) this.minecraft.getWindow().getGuiScaledWidth()
-					/ (double) this.minecraft.getWindow().getScreenWidth());
-			int j = (int) (this.minecraft.mouseHandler.ypos() * (double) this.minecraft.getWindow().getGuiScaledHeight()
-					/ (double) this.minecraft.getWindow().getScreenHeight());
-
-
-			GameRendererVRMixin.DATA_HOLDER.pumpkineffect = 0.0F;
```

Comment:
This is still needed, as right now, the gui does not get fixed when you take it off. (Or we fix it another way)